server/application/application.go

```diff
@@ -62,9 +63,10 @@ import (
 type AppResourceTreeFn func(ctx context.Context, app *appv1.Application) (*appv1.ApplicationTree, error)
 
 const (
-	maxPodLogsToRender                 = 10
-	backgroundPropagationPolicy string = "background"
-	foregroundPropagationPolicy string = "foreground"
+	maxPodLogsToRender                     = 10
+	backgroundPropagationPolicy     string = "background"
+	foregroundPropagationPolicy     string = "foreground"
+	maxGoroutinesForListApplication        = 50
```

Comment:
What was the logic for choosing 50 and not say 25, should this be user adjustable?